src/Microsoft.Health.Dicom.Core/Messages/ExtendedQueryTag/GetExtendedQueryTagErrorsResponse.cs

```diff
@@ -0,0 +1,30 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag;
+
+namespace Microsoft.Health.Dicom.Core.Messages.ExtendedQueryTag
+{
+    public class GetExtendedQueryTagErrorsResponse
+    {
+        public GetExtendedQueryTagErrorsResponse(
+            string tagPath,
+            IEnumerable<ExtendedQueryTagError> extendedQueryTagErrors
+            )
+        {
+            Path = tagPath;
+            ExtendedQueryTagErrors = extendedQueryTagErrors;
+        }
+
+        public string Path { get; set; }
+
+        // TODO put this in configs
+        public int PaginationOption { get; }
```

Comment:
I would leave this out right now if you don't know how this'll look. How do you imagine this looking? I don't see a type called `PaginationOption` in the PR